frontend/dynamic-demo-plugin/src/utils/bar.ts

```diff
@@ -1,8 +1,8 @@
-import { FeatureFlagHandler } from '@console/dynamic-plugin-sdk/src/extensions/feature-flags';
+import { SetFeatureFlag } from '@console/dynamic-plugin-sdk/src/extensions/feature-flags';
 
 export default (label: string) => `Hello ${label} Function!`;
 
-export const testHandler: FeatureFlagHandler = function() {
+export const testHandler: SetFeatureFlag = function() {
```

Comment:
This is already fixed in #8224 - please skip this change.

Console web app will build fine even with dynamic demo plugin containing errors.